lib/iris/cube.py

```diff
@@ -1054,6 +1132,25 @@ def coord_dims(self, coord):
 
         return matches[0]
 
+    def cell_measure_dims(self, cell_measure):
+        """
+        Returns a tuple of the data dimensions relevant to the given
+        CellMeasure.
+
+        * cell_measure
+            The CellMeasure to look for.
+
+        """
+        # Search for existing cell measure (object) on the cube, faster lookup
+        # than equality - makes no functional difference.
+        matches = [dims for cm_, dims in self._cell_measures_and_dims if
+                   cm_ is cell_measure]
```

Comment:
@marqh Okay, agreed ... but as a compromise I'd love for you to drop the appending of `_` to variable names, it's really not necessary here. Pretty please :pray: